### PR TITLE
[WIP] Docs: cli tour

### PR DESCRIPTION
**Pull request by kiwicopple**

### CODE REVIEW

File: web/docs/guides/local-development.mdx [hunk 1]
```diff
@@ -122,62 +130,76 @@ supabase stop
...
 
-If everything is working, you should have a React app running on `http://localhost:3000` and Supabase services running on `http://localhost:54321`.
+You should now see the contents of `boarders`. Hooray!
 
-When you're finished with Supabase, send <kbd>Ctrl-C</kbd> to stop the application and run
+## 3. Resetting schema changes
 
-```bash
-supabase stop   # Stop Supabase
+What if we ran some nasty SQL on the local database and want to wipe it all clean?
```
Comment (kiwicopple) [inline web/docs/guides/local-development.mdx:181]:
```suggestion
If you run some SQL on the local database which you want to revert, you can use the `reset` command.
```

File: web/docs/guides/local-development.mdx [hunk 2]
```diff
@@ -122,62 +130,76 @@ supabase stop
...
+
+```sql
+-- run on local database
+ALTER TABLE boarders ADD occupation text DEFAULT 'Hitchhiker';
```
Comment (kiwicopple) [inline web/docs/guides/local-development.mdx:185]: lowercase

File: web/docs/guides/local-development.mdx [hunk 3]
```diff
@@ -122,62 +130,76 @@ supabase stop
...
-This quick-start guide will show you, how you can configure a React app which uses the Supabase stack.
+You can also use the CLI to manage your migrations. 
 
-```bash
-# create a fresh React app
-npx create-react-app react-demo --use-npm
 
-# move into the new folder
-cd react-demo
+## 2. Changing schema
```
[the remark this answers is not in the document]
Comment (soedirgo) [reply web/docs/guides/local-development.mdx:138]: Hmm I guess "Making schema changes" is more apt

[the remark this answers is not in the document]
Comment (soedirgo) [reply web/docs/guides/local-development.mdx:138]:
```suggestion
## 2. Making schema changes
```

File: web/docs/guides/local-development.mdx [hunk 4]
```diff
@@ -122,62 +130,76 @@ supabase stop
...
 
-# set up Supabase
-supabase init
+To change the schema for the local database, simply run some SQL against the `DB URL` shown by `supabase start`. For the tour, we'll run these changes using a Postgres client of your choice:
 
-# Save the install supabase-js library
-npm install --save @supabase/supabase-js
+```sql
+CREATE TABLE boarders(
```
[the remark this answers is not in the document]
Comment (soedirgo) [reply web/docs/guides/local-development.mdx:143]: Obscure Hitchhiker's Guide reference, meant to be spaceship boarders. Can use a more neutral example if you like

**Review by soedirgo — APPROVED** [review]

**Pull request merged**